ci/dockerfiles/Dockerfile.alpine

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10.8-alpine
+
+RUN apk update & apk upgrade
+RUN apk add build-base
+
+RUN python -m pip install \
+  cython \
```

Comment:
Could we maintain the minimal set with files?

Reply:
Absolutely. One option is to create a `requirements-dev-minimal.txt` file in this CI folder, then can just execute a `COPY` command within the Dockerfile and install from that